src/plugins/intel_cpu/src/nodes/executors/eltwise.hpp

```diff
@@ -16,17 +16,23 @@ struct EltwiseAttrs {
     float alpha;
     float beta;
     float gamma;
+    enum BroadcastingPolicy {
```

Comment:
I would propose not to reuse BroadcastingPolicy from Eltwise node here. This is internal node property not designed as generic operation attribute. I can suggest just to have loop by inputs which will check that each input has equal shapes inside AclEltwiseExecutor::init